test/e2e/components/vl-cookie-consent.js

```diff
@@ -14,7 +14,7 @@ class VlCookieConsent extends VlElement {
     }
 
     async getOwner() {
-        const element = await this.shadowRoot.findElement(By.css('#owner'));
+        const element = await this.shadowRoot.findElement(By.css('[data-vl-owner]'));
```

Comment:
moeten er dat dan ook niet meerdere zijn?

Reply:
In de unit testen worden ze allemaal gecontroleerd, op e2e niveau lijkt dat me niet nodig?